ui/src/components/WorkerManagerWorkerTypesTable/index.jsx

```diff
@@ -0,0 +1,154 @@
+import React, { Component } from 'react';
+import { arrayOf, string } from 'prop-types';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+import Typography from '@material-ui/core/Typography/';
+import LinkIcon from 'mdi-react/LinkIcon';
+import memoize from 'fast-memoize';
+import { camelCase } from 'change-case';
+import { isEmpty, map, pipe, sort as rSort } from 'ramda';
+import { WorkerManagerWorkerTypeSummary } from '../../utils/prop-types';
+import DataTable from '../DataTable';
+import sort from '../../utils/sort';
+import Link from '../../utils/Link';
+import DateDistance from '../DateDistance';
+
+export default class WorkerManagerWorkerTypesTable extends Component {
+  static propTypes = {
+    workerTypes: arrayOf(WorkerManagerWorkerTypeSummary).isRequired,
+    searchTerm: string,
+    path: string,
```

Comment:
Remove `path`. Instead we could either use the `withRouter` decorator and extract the path from it or simply hard-code `/worker-manager` when performing the link.

Reply:
I don't like hard-coding things, that's why I passed down `path` in the first place. How is `withRouter` decorator better?

Reply:
With `withRouter`, there won't be any hard-coding. Feel free to look for the following links for an explanation:
* https://hackernoon.com/withrouter-advanced-features-of-react-router-for-single-page-apps-42b2a1a0d315
* https://itnext.io/basics-of-react-router-v4-336d274fd9e0 (search for withRouter)

Reply:
Oh, so we express the `withRouter` function as a decorator, ok 👌